examples/opensora_hpcai/scripts/infer_t5.py

```diff
@@ -10,6 +10,7 @@
 from tqdm import tqdm
 
 import mindspore as ms
+from mindspore._c_expression import ms_ctx_param
```

Comment:
add a FIXME tag here to remove of using internal API

Reply:
done